user_gen.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+
+
+def simp_poisson_process(rate=1, size=None):
+    sum_exp_draws = np.cumsum(np.random.exponential(scale=rate, size=size))
+    # sum_exp_draws = [sum(exp_draws[:i]) for i, x in enumerate(exp_draws)]
+    return sum_exp_draws
+
+
+def poisson_plus(lam=1, plus=1, size=None):
+    return np.random.poisson(lam=lam, size=size) + plus
+
+
+def int_exponential(rate=1, size=None):
+    exp_draws = np.random.exponential(scale=rate, size=size)
+    return np.round(exp_draws)
+
+
+def diffs_btw_exp(rate=1, size=None):
+    sorted_draws = sorted(np.random.exponential(scale=rate, size=size))
+    seq_diffs = [x-sorted_draws[i-1] for i, x in enumerate(sorted_draws)]
+    seq_diffs[0] = sorted_draws[0]
+    return np.array(seq_diffs)
+
+
+def is_pos_inc_seq(seq):
```

Comment:
would be nice to keep these hidden functions so that it's clear what should be exposed to the user (in this case us :-) )